test/e2e/suite/issuers/vault/issuer.go

```diff
@@ -18,6 +18,7 @@ package vault
 
 import (
 	"context"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Move to block below, as per https://cert-manager.io/docs/contributing/coding-conventions/#organizing-imports